test/boost/cdc_test.cc

```diff
@@ -1260,3 +1260,66 @@ SEASTAR_THREAD_TEST_CASE(test_change_splitting) {
         }
     }, mk_cdc_test_config()).get();
 }
+
+SEASTAR_THREAD_TEST_CASE(test_batch_with_row_delete) {
```

Comment:
It seems the test is in the patch before the change, I think the patches need to be reversed so the test is last?

Reply:
Happily, it's not true. See the real order of the branch -> https://github.com/jul-stas/scylla/commits/5709-cdc-preimage-on-row-delete--good

Reply:
Wow, that's the weirdest GH bug so far.